### PR TITLE
Added events for invalid messages

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

**Comment by palango** [comment]
Changes look good to me.

But this is adding new events, so we need to increase the `DB_VERSION`.

File: raiden/transfer/events.py [hunk 1]
```diff
@@ -583,6 +583,194 @@ def from_dict(
...
+            self.secrethash == other.secrethash and
+            self.reason == other.reason
+        )
+
+    def __ne__(self, other):
+        return not self.__eq__(other)
+
+    def to_dict(self) -> typing.Dict[str, typing.Any]:
+        result = {
+            'secrethash': self.secrethash,
```
Comment (palango) [inline raiden/transfer/events.py]: this needs to encode the `bytes`

File: raiden/transfer/events.py [hunk 2]
```diff
@@ -583,6 +583,194 @@ def from_dict(
...
```
Comment (palango) [inline raiden/transfer/events.py]: same here

File: raiden/transfer/events.py (hunk 1 above)
Comment (palango) [reply raiden/transfer/events.py]: like: `'secrethash': serialization.serialize_bytes(self.secrethash),`

**Comment by LefterisJP** [comment]
I am merging this since this is only adding new Raiden internal events emitted by the state machine in the unhappy path and has no additional logic changes.

These events will be very useful for debugging complicated problems. They have already helped a lot for the debugging of https://github.com/raiden-network/raiden/issues/2905 and as such I would want them for production.

**Pull request merged**